<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>takeGUI</class>
 <widget class="QMainWindow" name="takeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>撘乘公車</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTakeStep">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>撘乘站：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Take">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>朝陽科技大學</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDesStep">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目的地站：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Des">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>逢甲大學(福星路)</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_TakeInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <column>
     <property name="text">
      <string>路線</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>路線名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>方向</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>站序</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>中文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>英文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>經度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>緯度</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_To_TF_Info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>1200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <column>
     <property name="text">
      <string>路線</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>路線名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>方向</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>站序</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>中文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>英文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>經度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>緯度</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_TF_To_Info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>1200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <column>
     <property name="text">
      <string>路線</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>路線名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>方向</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>站序</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>中文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>英文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>經度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>緯度</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableDes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>1200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <column>
     <property name="text">
      <string>路線</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>路線名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>方向</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>站序</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>中文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>英文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>經度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>緯度</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="takeBTN">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查詢公車路線</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTakeBus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>撘乘站公車：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ToTF_Bus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>抵達轉乘站公車：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TF_To_Bus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>轉乘站公車：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Des">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目的地站下車：</string>
    </property>
   </widget>
   <widget class="QLabel" name="takeToDesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>970</y>
      <width>1280</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>撘乘資訊：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
